sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 # Release History
 
-## 4.0.0-beta.7 (Unreleased)
+## 4.0.0-beta.7 (2021-05-15)
 
+### New features
+
+- Added support for creating, updating and deleting custom role definitions.
+
+### Dependency Updates
+- Upgraded `azure-core` dependency to `1.16.0`
+- Upgraded `azure-core-http-netty` dependency to `1.9.2`
+- Upgraded `azure-core-http-okhttp` dependency to `1.6.2`
+- Upgraded `azure-identity` dependency to `1.3.0`
```

Comment:
nit: don't need to call these out as they are only test dependencies (azure-core-http-okhttp, azure-identity)